amical/mf_pipeline/bispect.py

```diff
@@ -977,27 +980,40 @@ def _compute_phs_error(complex_bs, fitmat, index_mask, npix, imsize=3):
     return phs_v2corr
 
 
-def _add_infos_header(infos, hdr, mf, pa, filename, maskname, npix):
+def _add_infos_header(infos, hdr, mf, pa, filename, maskname, npix, verbose=True):
```

Comment:
I don't think we should support warning deactivation, especially via boolean arguments (see for instance https://ariya.io/2011/08/hall-of-api-shame-boolean-trap)
```suggestion
def _add_infos_header(infos, hdr, mf, pa, filename, maskname, npix):
```

Reply:
I agree! I fixed this in a separate commit to remove the related `if` statement at the same time. Marking as resolved.